src/templates/csr/fabric-xl/index.svelte

```diff
@@ -28,6 +28,13 @@
 	export let MobileLayer2BackgroundPosition: GAMVariable;
 	export let MobileLayer3BackgroundImage: GAMVariable;
 	export let MobileLayer3BackgroundPosition: GAMVariable;
+	export let VideoURL: GAMVariable;
```

Comment:
We don't need to add these variables to the non video templates - we can just give them default values in the shared component